### PR TITLE
F/muwm 4767

### PR DESCRIPTION
**Pull request by nick-falco**
Add HR-Payroll card to the home page.

Fix for https://jira.cac.washington.edu/secure/RapidBoard.jspa?rapidView=1215&view=detail&selectedIssue=MUWM-4767

### CODE REVIEW

**Comment by nick-falco** [comment]
@charlon I added the following css for the Workday button, but you'll likely want to change it.

https://github.com/uw-it-aca/myuw/pull/2152/files#diff-a68c7fed035f5b1dc3f623f9546927cd770ea15f83df411d2f0d8e5d957766e2R7-R21

File: myuw/static/vue/components/_common/hr-payroll.vue [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    ...mapState({
+      showCard: function(state) {
+        if (this.isHomePage &&
+          state.user.affiliations.employee &&
+          !state.user.affiliations.student &&
+          !state.user.affiliations.instructor ||
+          state.user.affiliations.retiree ||
+          state.user.affiliations.past_employee) {
+          return true;
+        } else if (this.isAccountsPage &&
```
Comment (fanglinfang) [inline myuw/static/vue/components/_common/hr-payroll.vue:88]: You may skip "else" here.

File: myuw/static/vue/components/_common/hr-payroll.vue [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+               state.user.affiliations.stud_employee ||
+               state.user.affiliations.instructor) {
+          return true;
+        } else {
```
Comment (fanglinfang) [inline myuw/static/vue/components/_common/hr-payroll.vue:92]: Skip "else {", just return ...

Comment (fanglinfang) [reply myuw/static/vue/components/_common/hr-payroll.vue:92]: Looks good overall!

**Review by fanglinfang — APPROVED** [review]

**Review by SarthakSingh31 — APPROVED** [review]

**Pull request merged**